components/Pages/DocuSignPage/CartTotal.tsx

```diff
@@ -32,59 +32,51 @@ const CartTotal = () => {
           </h2>
           <hr className="border border-grey -mt-1" />
         </div>
-        <div className="flex flex-col gap-2 text-left font-share place-self-center">
+        <div className="flex flex-col gap-2 text-left font-share place-self-start">
           <div className="w-full flex justify-between gap-2 md:text-xl">
             <p>1x</p>
-            <p>Automized Contract</p>
+            <p className="md:text-[24px]">Automized Contract</p>
           </div>
         </div>
 
         <div className="flex flex-col gap-2 font-share text-[11px]">
           <div className="w-full flex items-center gap-2 ">
-            <p>Sub-Total</p>
+            <p className="md:text-[18px]">Sub-Total</p>
             <MoneyPara>2.00</MoneyPara>
           </div>
           <div className="w-full items-center flex gap-2">
-            <p>Taxes</p>
+            <p className="md:text-[18px]">Taxes</p>
             <MoneyPara>0.34</MoneyPara>
```

Comment:
For Dynamic Data:

Hardcoded values like 2.00, 0.34, and 2.14 may be more flexible if retrieved from a state or prop, allowing the component to handle dynamic pricing and taxes.